GoodFilmsApp/PosterHandler.cs

```diff
@@ -63,5 +59,88 @@ public PosterHandler(int numOfPictures, PosterBoxSettings s, ref GroupBox gb, IC
                 gb.Controls.Add(pb);
             }
         }
+
+        private void updateControls(List<FilmModel> films)
+        {
+            mv.Invoke(new Action(() => {
+                btnLeft.Enabled = page > 0;
+                btnRight.Enabled = btnRight.Enabled = films.Count >= size;
+                if (films.Count < size)
+                {
+                    lblStatus.Text = "page " + (page+1).ToString() + " / " + (getMaxOffset() / size).ToString();
```

Comment:
change this line to also add an extra page to the "out of how many pages" component

Reply:
this is only a simple cheeky fix though I I guess